### ISSUE TITLE
run the mod on the server?

### ISSUE THREAD

**Issue by mkrtes** [ezterry/DefaultWorldGenerator-port#4]
Please explain how I can run the mod on the server? I'm trying to make a water world, but generated a normal world. Here is the configuration
```
# Configuration file

general {
    # State of the Bonus Chest Button 0=disabled+shown, 1=enabled+shown, 2=disable+hidden, 3=enable+hidden [default: 0]
    I:"Bonus Chest State"=0

    # Copy the DefaultWorldData directory into the new world data directory [default: true]
    B:"Copy DefaultWorldData"=true

    # The world customization string (JSON, or super flat string) [default: ]
    S:CustomizationJson={"coordinateScale":684.412,"heightScale":684.412,"lowerLimitScale":512.0,"upperLimitScale":512.0,"depthNoiseScaleX":200.0,"depthNoiseScaleZ":200.0,"depthNoiseScaleExponent":0.5,"mainNoiseScaleX":80.0,"mainNoiseScaleY":160.0,"mainNoiseScaleZ":80.0,"baseSize":8.5,"stretchY":12.0,"biomeDepthWeight":1.0,"biomeDepthOffset":0.0,"biomeScaleWeight":1.0,"biomeScaleOffset":0.0,"seaLevel":256,"useCaves":true,"useDungeons":true,"dungeonChance":8,"useStrongholds":true,"useVillages":true,"useMineShafts":true,"useTemples":true,"useMonuments":true,"useMansions":true,"useRavines":true,"useWaterLakes":true,"waterLakeChance":4,"useLavaLakes":true,"lavaLakeChance":80,"useLavaOceans":false,"fixedBiome":0,"biomeSize":4,"riverSize":4,"dirtSize":33,"dirtCount":10,"dirtMinHeight":0,"dirtMaxHeight":256,"gravelSize":33,"gravelCount":8,"gravelMinHeight":0,"gravelMaxHeight":256,"graniteSize":33,"graniteCount":0,"graniteMinHeight":0,"graniteMaxHeight":80,"dioriteSize":33,"dioriteCount":0,"dioriteMinHeight":0,"dioriteMaxHeight":80,"andesiteSize":33,"andesiteCount":0,"andesiteMinHeight":0,"andesiteMaxHeight":80,"coalSize":17,"coalCount":20,"coalMinHeight":0,"coalMaxHeight":128,"ironSize":9,"ironCount":20,"ironMinHeight":0,"ironMaxHeight":64,"goldSize":9,"goldCount":2,"goldMinHeight":0,"goldMaxHeight":32,"redstoneSize":8,"redstoneCount":8,"redstoneMinHeight":0,"redstoneMaxHeight":16,"diamondSize":8,"diamondCount":1,"diamondMinHeight":0,"diamondMaxHeight":16,"lapisSize":7,"lapisCount":1,"lapisCenterHeight":16,"lapisSpread":16}

    # Prevent the user from changing the world type. [default: false]
    B:"Lock Worldtype"=true

    # The Suggested Seed for the pack, leave blank for a random seed, notethe player can change this even if locked [default: ]
    S:Seed=

    # The world generator to select by default [default: default]
    S:"World Generator"=customized
}


```

**Comment by ezterry** [comment]
The mod will only preset the server.properties file on the server, if you have a template already with the world generator/generator options set it will keep your existing settings.

Thus shutdown the server, delete the server.properties (or all lines not specific to your servers  customization) and the existing world before starting the server again.
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by ezterry** [comment]
Duplicate of #2
